apps/charterafrica/src/payload/fields/ecosystem/AirtableBaseSelect.js

```diff
@@ -0,0 +1,28 @@
+import { Select } from "payload/components/forms";
```

Comment:
@koech is it necessary to create custom components for select? Can the default payload select field work, all that is needed is to fetch the options.